model-optimizer/mo/back/offline_transformations.py

```diff
@@ -72,8 +72,9 @@ def read_model(path_to_xml):
     parser = argparse.ArgumentParser()
     parser.add_argument("--input_model")
     parser.add_argument("--framework")
-    parser.add_argument("--transform")
+    parser.add_argument("--transform", nargs='?', const='')
```

Comment:
How's that relevant?

Reply:
It's not relevant, anyway it is already not needed, will remove during rebase on latest master